pinot-hadoop/src/main/java/org/apache/pinot/hadoop/job/InternalConfigConstants.java

```diff
@@ -36,4 +36,10 @@
   public static final String PARTITION_FUNCTION_CONFIG = "partition.function";
 
   public static final String SORTED_COLUMN_CONFIG = "sorted.column";
+  public static final String ENABLE_PARTITIONING = "enable.partitioning";
+
+  // max records per file in each partition
+  public static final String PARTITION_MAX_RECORDS_PER_FILE = "partition.max.records.per.file";
+
+  public static final String PREPROCESS_NUM_FILES = "preprocess.num.files";
```

Comment:
Can you add comments on this field? This one has no effect if partitioning is enabled right?

Reply:
Please add comments